src/controllers/settings.test.js

```diff
@@ -0,0 +1,416 @@
+import test from 'ava'
+import * as td from 'testdouble'
+
+import { createResponse, createRequest } from 'node-mocks-http'
```

Comment:
Praise: Awesome that you got to make the tests work!

Reply:
Thanks! If you browse through them you can see I've disabled things here and there, but I think we can preserve a lot of the good work that was already done here.